packages/product-feed-google/easy-coding-standard.yml

```diff
@@ -1,6 +1,15 @@
 imports:
     - { resource: '%vendor_dir%/shopsys/coding-standards/easy-coding-standard.yml', ignore_errors: true  }
 
+services:
+    forbidden_private_visibility_fixer.google:
```

Comment:
I would like to have here a full package name to set some kind of standard (also in other product-feed-* packages)

Reply:
seems reasonable 👍